pallets/pallet-did-lookup/src/benchmarking.rs

```diff
@@ -241,6 +242,29 @@ benchmarks! {
 		assert!(ConnectedDids::<T>::get(&linkable_id).is_none());
 		assert!(ConnectedAccounts::<T>::get(did, linkable_id).is_none());
 	}
+
+	transfer_deposit {
+		let caller_0: T::AccountId = account("caller", 0, SEED);
+		let caller_1: T::AccountId = account("caller", 1, SEED);
```

Comment:
Same as above. Would love to see both callers to be more distinguishable.

Reply:
[b1dee26](https://github.com/KILTprotocol/kilt-node/pull/405/commits/b1dee2646b04e6f5e7481d7ac0960cf64b08eb26)